### PR TITLE
Add FAQ about CoinJoin via tBTC

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
NEW FAQ:
"I'd like to experience CoinJoin but I'm not comfortable with using real Bitcoin. What can I do?"

### CODE REVIEW

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -657,6 +657,16 @@ The coordinator sends this transaction over the tor network to a random full nod
 Wasabi is saving on mining fees by setting a confirmation target of roughly 12 hours, but you can re-register unconfirmed anonset outputs for the next round of CoinJoin.
 :::
 
+:::details
+### I'd like to experience CoinJoin but I'm not comfortable with using real Bitcoin. What can I do?
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
### I'd like to experience CoinJoin but I'm not comfortable using real Bitcoin. What can I do?
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -657,6 +657,16 @@ The coordinator sends this transaction over the tor network to a random full nod
...
+
+You can try to make a CoinJoin via Wasabi by using tBTC (testnet BTC).
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
You can try to make a CoinJoin via Wasabi on the Bitcoin TestNet (an alternative Bitcoin blockchain, to be used for testing).
Go to settings and change the network to TestNet.
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 3]
```diff
@@ -657,6 +657,16 @@ The coordinator sends this transaction over the tor network to a random full nod
...
+Just download Wasabi and and get free tBTC from faucets like:
+[Coinfaucet.eu](https://coinfaucet.eu/en/btc-testnet/)
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
[https://coinfaucet.eu/en/btc-testnet/](https://coinfaucet.eu/en/btc-testnet/)
```

File: docs/FAQ/FAQ-UseWasabi.md [hunk 4]
```diff
@@ -657,6 +657,16 @@ The coordinator sends this transaction over the tor network to a random full nod
...
+or
+[Mempool.co](https://testnet-faucet.mempool.co/)
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]:
```suggestion
[https://testnet-faucet.mempool.co/](https://testnet-faucet.mempool.co/)
```

**Comment by RiccardoMasutti** [comment]
Ready @MaxHillebrand @lontivero

File: docs/FAQ/FAQ-UseWasabi.md [hunk 5]
```diff
@@ -657,6 +657,17 @@ The coordinator sends this transaction over the tor network to a random full nod
 Wasabi is saving on mining fees by setting a confirmation target of roughly 12 hours, but you can re-register unconfirmed anonset outputs for the next round of CoinJoin.
 :::
 
+:::details
+### I'd like to experience CoinJoin but I'm not comfortable using real Bitcoin. What can I do?
+
+You can try to make a CoinJoin via Wasabi on the Bitcoin TestNet (an alternative Bitcoin blockchain, to be used for testing).
+Go to settings and change the network to TestNet.
+Just download Wasabi and and get free tBTC from faucets like:
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md]: It doesn't make sense to say `Just download Wasabi` after saying `Go to settings ...`
```suggestion
You can get tBTC from faucets like:
```

**Pull request merged**